src/JabbR-Core/Services/ChatService.cs

```diff
@@ -367,15 +368,17 @@ public void JoinRoom(ChatUser user, ChatRoom room, string inviteCode)
                 }
             }
 
+
             // Add this user to the room
+
             _repository.AddUserRoom(user, room);
 
             ChatUserPreferences userPreferences = user.Preferences;
             userPreferences.TabOrder.Add(room.Name);
             user.Preferences = userPreferences;
-
-            // Clear the cache
-            _cache.RemoveUserInRoom(user, room);
+            
```

Comment:
I updated the comment to remind us to add that line back in when the cache is working